src/pages/PassportRedemption/Passport/index.tsx

```diff
@@ -52,9 +45,7 @@ const Passport = (props: Props) => {
   useEffect(() => {
     if (id) {
       getContactInfo(id)
-        .then((contactInfo) => {
-          setShowInstagram(contactInfo.data.instagram);
-          setShowLyftRewardPromo(contactInfo.data.is_eligible_for_lyft_reward);
+        .then((res) => {
```

Comment:
Are we even using the API result from getContactInfo, if we aren't, why do we have this call?

Reply:
for now we dont have any connections to our new backend connected to lny as it is not fully implemented. I have a separate branch that has what the endpoints should look like, just not pushed yet since i dont have the local api setup yet so i ignored any changes to our interactions with the api in this branch

Reply:
Oh I see. Thanks!